firefox_puppeteer/ui/toolbars.py

```diff
@@ -31,6 +31,14 @@ def forward_button(self):
         return self.marionette.find_element(By.ID, 'forward-button')
 
     @property
+    def go_button(self):
+        """ Provides access to the go button from the navbar ui.
```

Comment:
nit: this button is inside the location bar. I think its fine to remove all behind `button`.